app/va/va_profile/va_profile_client.py

```diff
@@ -394,58 +259,6 @@ def _handle_exceptions(self, va_profile_id_value: str, error: Exception):
 
             raise exception from error
 
-    def _make_request(
```

Comment:
this functionality replaced by [get_profile](https://github.com/department-of-veterans-affairs/notification-api/blob/df5381ca4c7d32f1c224c819f84c86d5d8b68790/app/va/va_profile/va_profile_client.py#L72-L94)